Packs/Salesforce/Integrations/SalesforceFusionIAM/SalesforceFusionIAM.py

```diff
@@ -0,0 +1,345 @@
+
+        :type user_data: ``Dict[str, Any]``
+        :param user_data: User data in the application format
+
+        :return: An IAMUserAppData object that contains the data of the created user in the application.
+        :rtype: ``IAMUserAppData``
+        """
+        uri = URI_PREFIX + 'sobjects/FF__Key_Contact__c'
+        res = self._http_request(
+            method='POST',
+            url_suffix=uri,
+            json_data=user_data
+        )
+        user_app_data = res.json()
+        user_id = user_app_data.get('id')
+        username = user_app_data.get('userName')
+        is_active = True
+
+        return IAMUserAppData(user_id, username, is_active, user_app_data)
+
+    def update_user(self, user_id: str, user_data: Dict[str, Any]):
+        """ Updates a user in the application using REST API.
+
+        :type user_id: ``str``
+        :param user_id: ID of the user in the application
+
+        :type user_data: ``Dict[str, Any]``
+        :param user_data: User data in the application format
+
+        :return: An IAMUserAppData object that contains the data of the updated user in the application.
+        :rtype: ``IAMUserAppData``
+        """
+        uri = URI_PREFIX + f'sobjects/FF__Key_Contact__c/{user_id}'
+        params = {"_HttpMethod": "PATCH"}
+        self._http_request(
+            method='POST',
+            url_suffix=uri,
+            params=params,
+            json_data=user_data
+        )
+
+        return self.get_user_by_id(user_id)
+
+    def enable_user(self, user_id: str):
+        """ Enables a user in the application using REST API.
+        There is no action on the user for Salesforce Fusion instance needs to be taken.
+
+        :type user_id: ``str``
+        :param user_id: ID of the user in the application
+
+        :return: An IAMUserAppData object that contains the data of the user in the application.
+        :rtype: ``IAMUserAppData``
+        """
+
+        return self.get_user_by_id(user_id)
+
+    def disable_user(self, user_id: str) -> IAMUserAppData:
+        """ Removes a user in the application using REST API.
+
+        :type user_id: ``str``
+        :param user_id: ID of the user in the application
+
+        :return: An IAMUserAppData object that contains the data of the user in the application.
+        :rtype: ``IAMUserAppData``
+        """
+
+        uri = URI_PREFIX + f'sobjects/FF__Key_Contact__c/{user_id}'
+
+        self._http_request(
+            method='DELETE',
+            url_suffix=uri
+        )
+
+        return IAMUserAppData(user_id, "", False, {})
+
+    def get_app_fields(self) -> Dict[str, Any]:
+        """ Gets a dictionary of the user schema fields in the application and their description.
+
+        :return: The user schema fields dictionary
+        :rtype: ``Dict[str, str]``
+        """
+
+        uri = '/schema'                             # TODO: replace to the correct GET Schema API endpoint
```

Comment:
let's try this endpoint to get the schema: https://developer.salesforce.com/docs/atlas.en-us.230.0.api_rest.meta/api_rest/dome_sobject_describe.htm
replace "Account" with "FF__Key_Contact__c" and return {field.name: field.label} for field in fields (see the example response body)